schema/home/home_page_hero_units.js

```diff
@@ -0,0 +1,86 @@
+import cached from '../fields/cached';
+import gravity from '../../lib/loaders/gravity';
+import { GravityIDFields } from '../object_identification';
+import {
+  GraphQLEnumType,
+  GraphQLList,
+  GraphQLNonNull,
+  GraphQLObjectType,
+  GraphQLString,
+} from 'graphql';
+
+const HomePageHeroUnitType = new GraphQLObjectType({
+  name: 'HomePageHeroUnit',
+  fields: {
+    ...GravityIDFields,
+    cached,
+    heading: {
+      type: GraphQLString,
+    },
+    href: {
+      type: GraphQLString,
+      resolve: ({ link }) => link,
+    },
+    title: {
+      type: GraphQLString,
+      resolve: ({ mobile_title }) => mobile_title,
+    },
+    background_image_url: {
+      type: GraphQLString,
+      description: 'The image to show, on desktop this defaults to the wide version.',
+      args: {
+        version: {
+          type: new GraphQLEnumType({
+            name: 'HomePageHeroUnitImageVersion',
+            values: {
+              WIDE: {
+                value: 'wide',
+              },
+              NARROW: {
+                value: 'narrow',
+              },
+            },
+          }),
+        },
+      },
+      resolve: ({ platform, background_image_url, background_image_mobile_url }, { version }) => {
+        if (version) {
+          return version === 'wide' ? background_image_url : background_image_mobile_url;
```

Comment:
so this is if you wanted a narrow image on desktop or something? 

Because I'm just thinking of when there'd be a case when your platform would be such that you wanted the other image (you only have two options right? big/normal and small/mobile). These are just different ways of asking for one of two things :o.